packages/im-components/src/molecules/logo/logo.module.scss

```diff
@@ -0,0 +1,22 @@
+.image {
+  display: inline-block;
+  height: 100%;
+  padding: 0rem 2rem;
+}
+
+.logo {
+  display: inline-flex !important;
+  color: #000;
+  align-items: center;
+}
+
+.logo:hover {
+  color: #000
+}
+
+.text {
+  font-weight: bolder;
+  font-family: monospace;
```

Comment:
font-family should be of the entire app, then we can change for each section
are we using mono-space overall? if yes then it should be in app.css

Reply:
It is only for the logo. monospace is because we have appLoader with same font-family.